app/models/section.rb

```diff
@@ -25,18 +25,18 @@ def self.find(manual, section_uuid, published: false)
     if editions.empty?
       raise KeyError.new("key not found #{section_uuid}")
     else
-      Section.new(manual: manual, uuid: section_uuid, editions: editions)
+      Section.new(manual: manual, uuid: section_uuid, previous_edition: editions.first, latest_edition: editions.last)
     end
   end
 
   def_delegators :latest_edition, :title, :slug, :summary, :body, :updated_at, :version_number, :change_note, :minor_update, :exported_at
 
   attr_reader :uuid
 
-  def initialize(manual:, uuid:, editions:)
+  def initialize(manual:, uuid:, previous_edition: nil, latest_edition: nil)
     @slug_generator = SlugGenerator.new(prefix: manual.slug)
     @uuid = uuid
-    @editions = editions
+    @editions = [previous_edition, latest_edition].compact.uniq
```

Comment:
Can you explain why you needed to add the call to `Array#uniq`? Also are there any examples in the specs which check this behaviour?